File: androidApp/src/main/java/com/luciddreaming/android/ui/screens/AboutScreen.kt
package com.luciddreaming.android.ui.screens

import androidx.compose.foundation.layout.*
import androidx.compose.material3.*
import androidx.compose.runtime.*
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import androidx.compose.ui.unit.dp

@Composable
fun AboutScreen(
    onClose: () -> Unit
) {
    Column(
        modifier = Modifier
            .fillMaxSize()
            .padding(16.dp),
        verticalArrangement = Arrangement.Center,
        horizontalAlignment = Alignment.CenterHorizontally
    ) {
        Card(
            modifier = Modifier
                .fillMaxWidth()
                .padding(16.dp),
            elevation = CardDefaults.cardElevation(defaultElevation = 8.dp)
        ) {
            Column(
                modifier = Modifier
                    .fillMaxWidth()
                    .padding(16.dp)
            ) {
                Row(
                    modifier = Modifier
                        .fillMaxWidth()
                        .padding(bottom = 16.dp),
                    horizontalArrangement = Arrangement.End
                ) {
                    Button(onClick = onClose) {
                        Text("Close")
                    }
                }

                Text(
                    text = "About LucidDreaming",
                    style = MaterialTheme.typography.titleLarge,
                    modifier = Modifier.padding(bottom = 16.dp)
                )

                Text(
                    text = "LucidDreaming is a powerful tool for managing and monitoring Minecraft servers.",
                    modifier = Modifier.padding(bottom = 16.dp)
                )

                Text(
                    text = "Version: 1.0.0",
                    style = MaterialTheme.typography.bodyMedium,
                    modifier = Modifier.padding(bottom = 8.dp)
                )

                Text(
                    text = "Developed with Kotlin and Jetpack Compose",
                    style = MaterialTheme.typography.bodyMedium,
                    modifier = Modifier.padding(bottom = 16.dp)
                )

                Text(
                    text = "© 2026 LucidDreaming Team",
                    style = MaterialTheme.typography.bodySmall,
                    modifier = Modifier.align(Alignment.CenterHorizontally)
                )
            }
        }
    }
}

File: iosApp/LucidDreaming/UI/AboutScreen.swift
import SwiftUI

struct AboutScreen: View {
    @Environment(\.dismiss) private var dismiss
    
    var body: some View {
        NavigationView {
            VStack(spacing: 20) {
                Text("About LucidDreaming")
                    .font(.title)
                    .fontWeight(.bold)
                
                Text("LucidDreaming is a powerful tool for managing and monitoring Minecraft servers.")
                    .font(.body)
                    .multilineTextAlignment(.center)
                
                Text("Version: 1.0.0")
                    .font(.body)
                
                Text("Developed with Kotlin and SwiftUI")
                    .font(.body)
                
                Text("© 2026 LucidDreaming Team")
                    .font(.caption)
                    .padding(.top, 40)
            }
            .padding()
            .navigationBarItems(trailing: Button("Close") { dismiss() })
        }
    }
}
